Source/Find_Number_follow_Kaprekar_Constant.cpp

```diff
@@ -0,0 +1,156 @@
+#include <iostream>
+#include <vector>
+#include <climits>
+using std::swap;
+using std::vector;
+using std::cout;
+using namespace std;
+
+int partitionOfDigits(vector<long long int> &digitOfNumber, int lowestIndex, int highestIndex)
+{
+    int pivotPoint = digitOfNumber[highestIndex];
+    int iterationIndex1 = (lowestIndex - 1);
+    try
+    {
+        for (int iterationIndex2 = lowestIndex; iterationIndex2 <= highestIndex; iterationIndex2++)
+        {
+            if (digitOfNumber[iterationIndex2] < pivotPoint)
+            {
+                iterationIndex1++;
+                swap(digitOfNumber[iterationIndex1], digitOfNumber[iterationIndex2]);
+            }
+        }
+        swap(digitOfNumber[iterationIndex1 + 1], digitOfNumber[highestIndex]);
+    }
+    catch (...)
+    {
+        std::cerr << "This function is not accessible" << '\n';
+    }
+    return (iterationIndex1 + 1);
+}
+
+void quickSort(vector<long long int> &digitOfNumber, int lowestIndex, int highestIndex)
```

Comment:
This should be in different file, which will contains these kind of utility methods.